test/mksymbols.rb

```diff
@@ -42,4 +42,3 @@
 #{o}
 ];
 FOO
-
```

Comment:
Let's try to keep new lines at the end of files, see: http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline